controller/search.js

```diff
@@ -17,6 +17,13 @@ function setup( config, backend, query ){
       return next();
     }
 
+    // do not run controller if there are already results
+    // this was added during libpostal integration.  if the libpostal parse/query
+    // doesn't return anything then fallback to old search-engine-y behavior
+    if (res && res.hasOwnProperty('data') && res.data.length > 0) {
+      return next();
+    }
+
```

Comment:
I agree some logging would be nice. Note that we can't log queries themselves when users have DNT enabled. There is some code in the [middleware](https://github.com/pelias/api/blob/master/middleware/access_log.js#L32-L34) to help filter out sensitive query params when DNT is enabled that might be helpful.